public/app/plugins/panel/canvas/migrations.ts

```diff
@@ -66,6 +67,16 @@ export const canvasMigrationHandler = (panel: PanelModel): Partial<Options> => {
         }
       }
     }
+
+    // migrate oneClickLinks to oneClickMode
+    const root = panel.options?.root;
+    if (root?.elements) {
+      for (const element of root.elements) {
+        if (element.oneClickLinks) {
+          element.oneClickMode = OneClickMode.Link;
+        }
```

Comment:
This will ensure oneClickMode defaults to Off after migration.
```suggestion
        if (!element.oneClickMode) {
          element.oneClickMode = OneClickMode.Off;
        }
        if (element.oneClickLinks) {
          element.oneClickMode = OneClickMode.Link;
        }
```